### PR TITLE
Issue #8657: Modification of javadoc example markers to support metadata generation - javadoc

### PR DESCRIPTION
**Pull request by gaurabdg**
#8657

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocBlockTagLocationCheck.java [hunk 1]
```diff
@@ -72,6 +72,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <p>
  * To configure the default check:
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocBlockTagLocationCheck.java:78]: `To configure the default check:` let use it as one more marker

Comment (gaurabdg) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocBlockTagLocationCheck.java]: Done. Here is the metadata gen PR:
https://github.com/checkstyle/metadata-gen/pull/46

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTypeCheck.java [hunk 2]
```diff
@@ -112,6 +112,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <p>
  * To configure the default check:
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTypeCheck.java:118]: Please fix this all similar updates in this PR

Comment (gaurabdg) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTypeCheck.java]: Done

**Comment by romani** [comment]
@gaurabdg ,

https://travis-ci.org/github/checkstyle/checkstyle/jobs/717240635#L796
please investigate and fix
```
----------
7961. ERROR in /home/travis/build/checkstyle/checkstyle/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java (at line 57)
797	* To configure the default check that will check {@code @param}, {@code
798	                                                                 ^^^^^^
799Javadoc: Missing closing brace for inline tag
```

**Comment by gaurabdg** [comment]
> @gaurabdg ,
> 
> https://travis-ci.org/github/checkstyle/checkstyle/jobs/717240635#L796
> please investigate and fix
> 
> ```
> ----------
> 7961. ERROR in /home/travis/build/checkstyle/checkstyle/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java (at line 57)
> 797	* To configure the default check that will check {@code @param}, {@code
> 798	                                                                 ^^^^^^
> 799Javadoc: Missing closing brace for inline tag
> ```

@romani Done

**Pull request merged**